examples/blueprint-construct/index.ts

```diff
@@ -41,14 +42,19 @@ export default class BlueprintConstruct {
             iam.ManagedPolicy.fromAwsManagedPolicyName("AmazonSSMManagedInstanceCore")
         ]);
 
+        const ampWorkspaceName = "blueprints-amp-workspace";
+        const ampPrometheusEndpoint = (blueprints.getNamedResource(ampWorkspaceName) as unknown as amp.CfnWorkspace).attrPrometheusEndpoint;
```

Comment:
Please see my comments in the docs on the style of referencing an attribute from a resource, provided by a resource provider

Reply:
fixed,